pipes/config.yaml

```diff
@@ -192,6 +192,9 @@ vphaser_max_bins: 10
 # allele freq > 0.005. If false, no filtering is performed.
 vcf_merge_naive_filter: false
 
+# Random seed, for non-deterministic steps.  0 means use current time.
+random_seed: 23923937
```

Comment:
Hm... did you assign a deterministic number here by default in order to make testing more deterministic? I think this file is often used as a template for user config files, so maybe we should make a config-tests.yaml that gets included only when running tests.

Reply:
I just thought, users want determinism.  But maybe not, since many tools use time as seed by default.  Btw, mummer also makes random choices, with no way to set the seed.  For testing, LD_PRELOAD of srand() can be used to derandomize most programs.

Reply:
There's a ton of tools that make random choices, and many don't offer a way to fix the seed.. early versions of BWA were definitely like this (maybe even current ones). We use the `Random` option on calls to Novoalign. Let's default to zero?